Set.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Zadanie1
+{
+    public class Set
```

Comment:
Ta klasa to raczej "Range" a nie "Set"

("Set" ma dość wąsko określone znaczenie)

Nazywanie to najważniejsza rzecz w programowaniu 😸